pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -1,10 +1,16 @@
+import numpy as np
 import pytest
 
-import pandas.util._test_decorators as td
+from pandas.compat._optional import import_optional_dependency
 
 import pandas as pd
 import pandas._testing as tm
 
+try:
+    pa = import_optional_dependency("pyarrow", min_version="0.16.0")
+except ImportError:
+    pytestmark = pytest.mark.skip(reason="Pyarrow not available")
```

Comment:
To avoid the imports in every test (as you did now), you could also do a 

```
try:
    import pyarrow as pa
except ImportError:
    pa = None
```